src/gfx/vid_file.cpp

```diff
@@ -104,10 +104,8 @@ class VidFileImpl
                         pCodecCtx, pNextFrameYUV, &frameFinished,
                         packetBuffer, packetBufferSize );
                     #endif
-                    VSFileSystem::vs_dprintf(3, "dts %lld: Decoded %d bytes %s\n", 
-                        int64_t(packet.dts),
-                        int(bytesDecoded),
-                        (frameFinished ? "Got frame" : "")
```

Comment:
vid_file.cpp as a whole doesn't compile with current versions of FFMPEG et al, so I've had to disable FFMPEG. As a result, this file hasn't actually been compiled or tested. However, my IDE (Visual Studio Code) did have syntax highlighting enabled for this file, and I think it would have caught most common types of errors.